.github/ISSUE_TEMPLATE/a_issue_report.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Issues
+about: Create an issue if you need any help
+title: '[Issue] '
+labels: 'help wanted, question'
+assignees: ''
+
+---
+
+**Describe the issue**
+<!--- A clear and concise description of what the issue is. -->
+
+**Which build did you use?**
```

Comment:
Change to `branch` or `commit` maybe?

Reply:
yeah i was thinking about this. `branch` might be better